src/internal/views/nunjucks/internal-search/index.njk

```diff
@@ -53,7 +53,7 @@
 {% endmacro %}
 
 {% macro documentLink(doc) %}
-  <a href="/{{ "expired-" if not doc.isCurrent }}licences/{{ doc.documentId }}">
+  <a href="/licences/{{ doc.licence.id }}">
```

Comment:
I would suggest wrapping this <a> ref in a conditional that checks for the licence.id being "there" first. E.g.

```
{% if doc.licence.id %}
<a...... >{{ doc.licenceNumber}}</a>
{% elseif %}
{{ doc.licenceNumber}}
{% endif %}
```

Purely because when I was poking around locally, I did find a few random results where the licence ID wasn't there and as a result, tapping on the link just took me to /licences

Reply:
Done